.github/workflows/test-load-k6.yaml

```
@@ -0,0 +1,64 @@
+name: k6 Load Test
+
+on:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+
+jobs:
+  run-tests:
+    name: Run k6 tests
```

```suggestion
    name: Run tests
```

I like the idea of dropping "k6" from everything so it reads more generically long-term should there be a change. The filename can change too.